grafana-provisioning/dashboards/Artemis__BGP_Updates_per_service.json

```diff
@@ -0,0 +1,421 @@
+          ],
+          "metricColumn": "prefix",
+          "rawQuery": true,
+          "rawSql": "SELECT\n  floor(extract(epoch from \"timestamp\")) AS \"time\",\n  service AS metric,\n  count(*) as \"update\"\nFROM bgp_updates\nWHERE\n  $__timeFilter(\"timestamp\") AND type = 'W'\nGROUP BY 1,2\nORDER BY 1,2",
+          "refId": "A",
+          "select": [
+            [
+              {
+                "params": [
+                  "peer_asn"
+                ],
+                "type": "column"
+              },
+              {
+                "params": [
+                  "count"
+                ],
+                "type": "aggregate"
+              },
+              {
+                "params": [
+                  "peer_asn"
+                ],
+                "type": "alias"
+              }
+            ]
+          ],
+          "table": "bgp_updates",
+          "timeColumn": "\"timestamp\"",
+          "timeColumnType": "timestamp",
+          "where": [
+            {
+              "name": "$__timeFilter",
+              "params": [],
+              "type": "macro"
+            },
+            {
+              "datatype": "inet",
+              "name": "",
+              "params": [
+                "prefix",
+                "IN",
+                "$prefix"
+              ],
+              "type": "expression"
+            }
+          ]
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "Withdrawals",
+      "transparent": true,
+      "type": "bargauge"
+    },
+    {
+      "datasource": "ArtemisPSQL",
+      "gridPos": {
+        "h": 23,
+        "w": 12,
+        "x": 12,
+        "y": 25
+      },
+      "id": 16,
+      "links": [],
+      "options": {
+        "displayMode": "gradient",
+        "fieldOptions": {
+          "calcs": [
+            "sum"
+          ],
+          "defaults": {
+            "decimals": 0,
+            "mappings": [],
+            "max": 100,
+            "min": 0,
+            "thresholds": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "green",
+                "value": 80
+              }
+            ],
+            "unit": "none"
+          },
+          "override": {},
+          "values": false
+        },
+        "orientation": "horizontal"
+      },
+      "pluginVersion": "6.4.1",
+      "targets": [
+        {
+          "format": "time_series",
+          "group": [
+            {
+              "params": [
+                "$__interval",
+                "none"
+              ],
+              "type": "time"
+            }
+          ],
+          "metricColumn": "prefix",
+          "rawQuery": true,
+          "rawSql": "SELECT\n  floor(extract(epoch from \"timestamp\")) AS \"time\",\n  service AS metric,\n  count(*) as \"update\"\nFROM bgp_updates\nWHERE\n  $__timeFilter(\"timestamp\") AND type = 'A'\nGROUP BY 1,2\nORDER BY 1,2",
```

Comment:
Could you also include 'W's? (not only announcements)

Reply:
It's already in that dashboard on a separate bar gauge. It's broken down into total, withdrawals and announcements.
https://github.com/FORTH-ICS-INSPIRE/artemis/pull/251/files/65faaf4d0e7efe3025326be32c32f25e610c1d4d#diff-c4ecf84e9654e38e3c16df7a9818390dR223

I could fix the graph headers, instead of add make it announcements. Should I proceed?

Reply:
Sorry I missed the other bar gauge while reviewing the json. It is fine then. Regarding the headers, yes this is fine, feel free to fix.
As discussed on discord, please also make a small wiki page if possible with the current steps to access and use the new dashboards (what  dashboards are available, at what intervals, etc.). Thanks again for everything!

Reply:
https://github.com/FORTH-ICS-INSPIRE/artemis/wiki/Grafana-Dashboards---Timeseries

Reply:
Pushing changes on github and will write the wiki page later in the afternoon. I thank you for this great project!